test/test_fill_database.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
I think it still applies; the end goal is still to fill the Presto "database". Or we could rename it to `fill_crash_aggregates.py`, which might be a bit better.